index.html

```diff
@@ -63,6 +63,236 @@ <h1>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="works_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+        <div class="grid-container">
+          <img
+            class="image-fluid"
+            src="images/works/image_3.png"
+            alt="background_image"
+          />
+          <div class="text">
+            <span class="tonic">Tonic</span>
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="works_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+        <div class="grid-container">
+          <img
+            class="image-fluid space-sm"
+            src="images/works/image_4.png"
+            alt="background_image"
+          />
+          <div class="text">
+            <span class="multi-post"
+              >Multi-Post<br />
+              Stories</span
+            >
+            <ul class="flex-items">
+              <li class="list-item canopy">CANOPY</li>
+              <li class="list-item role">Back End Dev</li>
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="works_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="vertical-space"></div>
```

Comment:
- [ ] As I mentioned, this is not the proper way of adding a space between two elements. Please remove this div and refer to what I mentioned above in order to add space between this section and others. **HINT:** you can try adding `margin-bottom` to the section with `class='work'`

Reply:
I have removed the white space as advised earlier.